Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -16,23 +17,60 @@ public class ReportService : IReportService
         public void UpdateDashboardData(long instanceId)
         {
             UpdateAtlassianUsers(instanceId);
```

Comment:
Let's rename this methods. Instead of Update let's use Sync. I think it is more suggestive. In a lot of cases we will also handle deleted items, not just updates.